charts/ccd-case-management-web/requirements.yaml

```diff
@@ -2,3 +2,8 @@ dependencies:
   - name: nodejs
     version: 1.0.2
     repository: '@hmcts'
+  - name: idam-pr
```

Comment:
Can you also bump this chart version up in Chart.yaml please

Reply:
I actually meant the ccd-case-management-web chart version, but the bump you did was a good bump. but can you update Chart.yaml too please

Reply:
Thanks. Actually i didn't read full comment. Bumped now :)